praetorian_cli/sdk/test/ui/test_aegis_info.py

```
@@ -0,0 +1,44 @@
+
+
+class Menu(MockMenuBase):
+    def __init__(self, agent=None):
+        super().__init__()
+        self.selected_agent = agent
+
+
+def test_info_no_selected_agent():
+    menu = Menu(agent=None)
+    handle_info(menu, [])
+    assert any("No agent selected" in l for l in menu.console.lines)
```

Ruff (0.12.2)

Ambiguous variable name: `l`

(E741)